### ISSUE TITLE
Unable to get a google drive photo from the library

### ISSUE THREAD

**Issue by virt87** [filestack/filepicker-android#95]
When the Google drive app is allowed to get images from the camera we are able to see that google drive images in Gallery.
![device-2016-04-08-230616](https://cloud.githubusercontent.com/assets/1911418/14388005/89d7dad2-fdde-11e5-993e-0ce8471459c1.png)

The problem occurs in FilesUtils.java class
```
public static TypedFile getTypedFileFromUri(Context context, Uri uri) {
        String mimetype;
        if("content".equalsIgnoreCase(uri.getScheme())) {
...
File file = new File(path); // here the file will be nil, as the returned path being = null

```

The biggest problem resides in getPath. In
```
 if (isKitKat && DocumentsContract.isDocumentUri(context, uri)) {
```
there is no option for check against  com.google.android.apps.docs and in this way null is returned, giving an error when retrieving a path to the file from the provided URI

**Comment by virt87** [comment]
Any update?

**Comment by janhudzik** [comment]
Fixed the issue by merging pull request https://github.com/filestack/filepicker-android/pull/96. Thank you for your contribution.

Fix will be available in 3.10.0 release.
